--- solarized.vim.org	2016-12-10 09:31:10.521397000 +0900
+++ solarized.vim	2016-12-10 09:33:26.734872100 +0900
@@ -61,11 +61,11 @@ if &background ==# 'light'
 endif
 
 let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
-let s:p.normal.left = [ [ s:base03, s:blue ], [ s:base03, s:base00 ] ]
+let s:p.normal.left = [ [ s:base03, s:green ], [ s:base03, s:base00 ] ]
 let s:p.normal.right = [ [ s:base03, s:base1 ], [ s:base03, s:base00 ] ]
 let s:p.inactive.right = [ [ s:base03, s:base00 ], [ s:base0, s:base02 ] ]
 let s:p.inactive.left =  [ [ s:base0, s:base02 ], [ s:base0, s:base02 ] ]
-let s:p.insert.left = [ [ s:base03, s:green ], [ s:base03, s:base00 ] ]
+let s:p.insert.left = [ [ s:base03, s:blue ], [ s:base03, s:base00 ] ]
 let s:p.replace.left = [ [ s:base03, s:red ], [ s:base03, s:base00 ] ]
 let s:p.visual.left = [ [ s:base03, s:magenta ], [ s:base03, s:base00 ] ]
 let s:p.normal.middle = [ [ s:base1, s:base02 ] ]
